docs/pages/database-access/guides/redshift-serverless.mdx

```diff
@@ -0,0 +1,400 @@
+
+Remember to replace the example AWS account ID.
+
+Create and apply a permission policy to allow access to Redshift Serverless.
+This policy can allow access to all instances:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "redshift-serverless:GetCredentials",
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+Or you can restrict the `Resource` line to a specific Redshift Serverless workgroup:
+
+```json
+{
+...
+      "Resource": "arn:aws:redshift-serverless:us-west-2:1234567890:workgroup/some-workgroup-id"
+...
+}
+```
+
+## Step 3/5. Create a Teleport role for Redshift Serverless access
+
+On your workstation logged in to your Teleport cluster with `tsh`, define a new
+role to provide access to Redshift Serverless. Our example file is
+`redshift-role.yaml`:
+
+```yaml
+version: v5
+kind: role
+metadata:
+  name: redshift-serverless-access
+spec:
+  allow:
+    db_labels:
+      '*': '*'
+    db_names:
+    - dev
+    db_users:
+    - 'teleport-redshift-serverless-access' # only the role name, do not specify the full AWS ARN
+```
+
+- The value of `db_users` corresponds to the IAM role created in the previous step.
+- The value(s) for `db_names` will depend on your Redshift Serverless configuration,
+  but `dev` is the default name applied by AWS. You can also provide `*` to grant
+  access to all instances.
+
+Save this file and apply it to your Teleport cluster:
+
+```code
+$ tctl create -f redshift-role.yaml
+role 'redshift-serverless-access' has been created
+```
+
+Add this role to your Teleport user. You can do this through the web UI, or by
+downloading your user definition:
+
+```code
+$ tctl get users/yourUser > yourUser.yaml
+```
+
+Adding the role:
+
+```yaml
+...
+  roles:
+  - redshift-serverless-access
+...
+```
+
+And applying the changes:
+
+```code
+$ tctl create -f yourUser.yaml
+user "yourUser" has been updated
+```
+
+## Step 4/5. Install and start the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+On the same host, create a Teleport configuration file:
+
+```code
+$ teleport db configure create \
+   -o file \
+   --proxy=teleport.example.com:443 \
+   --token=/tmp/token \
+   --redshift-serverless-discovery=us-west-1
+```
+
+- Replace `teleport.example.com` with your Teleport proxy URI or cloud tenant
+  address (e.g. `mytenant.teleport.sh`).
+- Provide the token from the previous step to the `--token` flag.
+- Update the value of `--redshift-serverless-discovery` to match your AWS region.
+
+This command will generate a Database Service configuration with Redshift
+Serverless auto-discovery enabled and place it at `/etc/teleport.yaml`.
+
+<Admonition type="tip" title="Filter with AWS tags">
+Use `--aws-tags <key>=<value>,<key>=<value>` to only match those with specified
```

Comment:
What does "those" refer to?

Reply:
I _think_ in this instance "those" would be "Redshift Serverless workgroups", but would like @greedy52 to confirm before I change it.

Reply:
yes, "Redshift Serverless workgroups" in this context. Or "AWS resources" in general, as it applies to other discoveries.

Reply:
Updated 👍